### PR TITLE
B-02032 ui cleanup

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/holofuel/components/Header/Header.js [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
   if (agentLoading) agentLoading = <h4>Loading...</h4>
 
   return <header>
     <section styleName='header'>
       <div styleName='left-nav'>
-        {leftNav}
+        <MenuButton onClick={hamburgerClick} inboxCount={inboxCount} />
       </div>
       <div styleName='center-nav'>
-        <div styleName={cx('page-header', { desktop: isWide })}>
```
Comment (JettTech) [inline src/holofuel/components/Header/Header.js:24]: Was reversal the desktop version an additional verbal request?  I don't see anything about removing/reversing the desktop-style side menu in the 'clean-up ui' ticket description.

File: src/holofuel/components/SideMenu/SideMenu.js [hunk 2]
```diff
@@ -62,15 +59,6 @@ export default function SideMenu ({
               Profile
             </Link>
           </li>
-          <li>
-            <div styleName='loading-row'>
```
Comment (JettTech) [inline src/holofuel/components/SideMenu/SideMenu.js:66]: The 'loading-row' styles will need to be applied to the previous page link (...and probably renamed), in order to maintain the divider style, separating the holofuel pages from the return to hp-admin link.

File: src/holofuel/components/SideMenu/SideMenu.js [hunk 3]
```diff
@@ -5,9 +5,8 @@ import { Link, useLocation } from 'react-router-dom'
 import HashAvatar from 'components/HashAvatar'
 import { presentAgentId } from 'utils'
 import CopyAgentId from 'holofuel/components/CopyAgentId'
-import Button from 'components/UIButton'
-import Loading from 'components/Loading'
 import BackIcon from 'components/icons/BackIcon'
+import MenuButton from 'holofuel/components/MenuButton'
```
Comment (JettTech) [inline src/holofuel/components/SideMenu/SideMenu.js:9]: In Firefox, the menuButton has a bit of an alignment bug.  Here's a screenshot for visual clarity:
image

**Review by JettTech — CHANGES_REQUESTED** [review]
Overall, good, just one small alignment issue to resolve, and a couple questions.

File: src/holofuel/components/SideMenu/SideMenu.js (hunk 3 above)
Comment (robbiecarlton) [reply src/holofuel/components/SideMenu/SideMenu.js:9]: good catch

File: src/holofuel/components/Header/Header.js (hunk 1 above)
Comment (robbiecarlton) [reply src/holofuel/components/Header/Header.js:24]: Yeah it is, just didn't make it into the ticket

**Review by JettTech — APPROVED** [review]

**Comment by JettTech** [comment]
Verified in Firefox.. LGTM! :shipit:

**Pull request merged**